### PR TITLE
[Author] 2 yr mean citedness; author institution

### PR DESCRIPTION
**Pull request by yattias**
- Add `AuthorInstitution` model
- Add "impact factor" also known as 2 year mean citedness
- Remove `AuthorCitation` model

### CODE REVIEW

File: src/paper/openalex_util.py [hunk 1]
```diff
@@ -156,3 +159,40 @@ def process_openalex_works(works):
     for paper_id, topics_and_concepts in paper_to_topics_and_concepts.items():
         openalex_concepts, openalex_topics = topics_and_concepts
         create_paper_related_tags(paper_id, openalex_concepts, openalex_topics)
+
+    openalex_authorships = work.get("authorships")
+
+    print("work:", work)
```
Comment (gzurowski) [inline src/paper/openalex_util.py:165]: Is that a leftover?

File: src/user/related_models/author_model.py [hunk 2]
```diff
@@ -9,6 +9,7 @@
 from paper.utils import PAPER_SCORE_Q_ANNOTATION
 from purchase.related_models.purchase_model import Purchase
 from researchhub_case.constants.case_constants import APPROVED
+from user.related_models.author_institution import AuthorInstitution
```
Comment (gzurowski) [inline src/user/related_models/author_model.py:12]: Nit: Is that change needed?

**Review by gzurowski — APPROVED** [review]
LGTM 👍

File: src/user/related_models/author_model.py (hunk 2 above)
Comment (yattias) [reply src/user/related_models/author_model.py:12]: Yes, it is needed because there is no `_init.py` file in that directory which includes the model. Django is unable to "see" it. I could alternatively create an _init file but didn't feel it was needed
